R/arima.R

```diff
@@ -462,6 +462,8 @@ specials_arima <- new_specials(
 #' }
 #' }
 #' 
+#' If the RHS of `formula` is left blank, `ARIMA` will use a default search space of `pdq() + PDQ()`, ie, fit a model with seasonal and nonseasonal terms (plus a mean). Note that a seasonal model requires at least 2 full seasons' worth of data; if this is not available, `ARIMA` will revert to a nonseasonal model with a warning.
```

Comment:
I don't think this comment belongs in the `PDQ()` specials subsection.

Creating a new section for automatic model selection above the `#' @section Specials:` would be easier for users to find. Further, you could mention that automatic selection searches the specified `pdq()`,`PDQ()` **and intercept** inputs for the best model (lowest `ic`) that also satisfies the `order_constraint`.
It is also worth noting that non-seasonal automatic selection can be done with specifying only `PDQ(0,0,0)`.

When specifying a model fully (avoiding automatic selection), the intercept and `pdq()`/`PDQ()` values must be specified. For example, `~ 1 + pdq(1,1,1) + PDQ(0,1,1)`.